src/lib/pages/workflows-with-new-search.svelte

```diff
@@ -266,7 +258,11 @@
             />
           {/if}
         </p>
+        <div class="h-1 w-1 rounded-full bg-gray-400" />
       {/if}
+      <Button variant="link" on:click={() => exportWorkflows($workflows)}
+        >Download JSON</Button
```

Comment:
What do you think about hiding or disabling this if there are no Workflows?

![Screenshot 2023-07-18 at 3 07 25 PM](https://github.com/temporalio/ui/assets/15069288/83fdbace-e968-416f-989b-ac9a22f328f9)

Reply:
Yeah probably a good idea, it would just download an empty array in this case. Curious though, how did you get this to not show 0 workflows count to the left of Download JSON?

Reply:
Cropped out the important part of that screenshot, that state was when I was on a Namespace that didn't exist. Looks like with your recent changes the `Download JSON` is disabled for it.

![Screenshot 2023-07-19 at 9 19 26 AM](https://github.com/temporalio/ui/assets/15069288/661f490e-485d-42f9-bca0-90c1f4125cb2)